clang/test/Preprocessor/riscv-target-features.c

```diff
@@ -116,6 +116,7 @@
 
 // CHECK-NOT: __riscv_smaia {{.*$}}
 // CHECK-NOT: __riscv_ssaia {{.*$}}
+// CHECK-NOT: __riscv_smepmp {{.*$}}
```

Comment:
This is the experimental extension section according to the comment on 115. I guess we're bad at moving things out of this section when they go non-experimental

Reply:
It's fixed now.